### DISCUSSION TITLE
Is it possible to have a setup with css modules instead of emotion?

### DISCUSSION THREAD

**Discussion by roman-yakobnyuk** [belgattitude/nextjs-monorepo-example#1940] [General]
I would love to generate my styles statically while still having the benefit of ui-lib.
I can get it to work with CSS-in-JS solutions, but I would love to see an example with CSS modules in ui-lib folder.

How would you do that? Would folder sync be an acceptable solution, similar to assets?

Currently, nextjs wont let me import CSS modules from ui-lib due to "Not importing CSS modules from node_modules" folder.

I know there is a transpilation lib that can be applied, but that one is not very stable and something I would like to avoid towards simpler and more stable solutions.

**Comment by belgattitude** [comment]
Yes it is a limitation of nextjs. I can't really help on that, but I would try to find an issue or discussion I guess they'll take your comment into consideration. Not sure it's a hard limit.

That said, have a look to vanilla-extract. ccs-modules are nice, but not sure if I would try to make a design-system based on it.
